package.json

```diff
@@ -34,7 +34,7 @@
     "ts-eager": "2.0.2",
     "ts-jest": "29.1.0",
     "turbo": "1.13.3",
-    "typescript": "4.9.5"
+    "typescript": "5.5.2"
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** ‏revert

We try to keep the typescript version consistent across the monorepo. It's not 100% consistent right now, but that is an explicit goal.

<img width="558" alt="Screenshot 2024-06-28 at 10 32 50 AM" src="https://github.com/vercel/vercel/assets/41545/21fc4406-18b1-4498-9aea-fd4f2a00c343">

If we need to make this update, can do we it across the whole monorepo in a separate PR? Happy to fast track reviews on that if tests pass.